src/ralph/reports/views.py

```diff
@@ -173,6 +232,106 @@ def prepare(self, model, dc=None):
             )
 
 
+class BaseRelationsReport(BaseReport):
+    template_name = 'reports/report_relations.html'
+    with_modes = True
+    links = False
+
+    def prepare(self, model, *args, **kwargs):
+        pass
+
+    def get_response(self, request, result):
+        data = tablib.Dataset(*result, headers=self.headers)
+        response = HttpResponse(
+            data.csv,
+            content_type='text/csv;charset=utf-8'
+        )
+        response['Content-Disposition'] = 'attachment;filename={}'.format(
+            self.filename
+        )
+        return response
+
+
+class AssetRelationsReport(BaseRelationsReport):
+    slug = 'asset-relations'
+    name = _('Asset - relations')
+    description = _('Asset list of information about the user, owner, model.')
+    filename = 'asset_relations.csv'
+    headers = ASSETS_COLUMNS
+
+    def get_assets_rows(self, model):
+        queryset = model.objects.all()
+        for asset in queryset:
+            row = [getattr_dunder(asset, column) for column in ASSETS_COLUMNS]
```

Comment:
how many SQLs are executed during generating this report? wouldn't it be better to pass these fields to `values_list` and do everything on SQL level?